subber/reddit.py

```diff
@@ -72,6 +75,18 @@ def get_user_recommendations(session, user):
                              '{}'.format(user, sim_user))
             logger.exception(e)
 
+    # if configuration indicates we could not recommend active subs
+    if cfg['recommend-actives'] == 'False':
```

Comment:
This seems a bit loose.. it would be safer to use [truth testing](https://docs.python.org/3.6/library/functions.html#bool) so that both `false` and `False` work.

Unfortunately, the config file loading logic is limited (i.e. it is set up to read `string` values even though config parser supports reading `bool` values); I will file an issue for this to be addressed in a future PR.